test/Amazon.SecretsManager.Extensions.Caching.Tests/Amazon.SecretsManager.Extensions.Caching.Tests.csproj

```diff
@@ -1,19 +1,22 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
-
-  <PropertyGroup>
-    <TargetFrameworks>net461</TargetFrameworks>
-    <IsPackable>false</IsPackable>
-  </PropertyGroup>
-
-  <ItemGroup>
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.9.0" />
-    <PackageReference Include="Moq" Version="4.10.1" />
-    <PackageReference Include="xunit" Version="2.4.0" />
-    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.0" />
-  </ItemGroup>
-
-  <ItemGroup>
-    <ProjectReference Include="..\..\src\Amazon.SecretsManager.Extensions.Caching\Amazon.SecretsManager.Extensions.Caching.csproj" />
-  </ItemGroup>
-
-</Project>
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFrameworks>net6.0;net5.0;netcoreapp3.1;netcoreapp2.1;net461</TargetFrameworks>
```

Comment:
`netstandard2.0` is not a runtime but a collection of runtimes. For a binary project like the tests, we need to provide the specific runtimes we want to test against. This covers all the popular runtimes and .NET 6 (preview)